dangvuthang/COSC2769-rmit
# Feat/integrate google login

### Changelog
Introducing new login mechanism with google

### How to test

1. Start both frontend and backend app
2. Navigate to /login, or /signup page
3. Click on the `Login with Google` button
4. See if the account is correctly created once finished
